### ISSUE TITLE
minor issues in doc examples

### ISSUE THREAD

**Issue by tmtron** [josdejong/mathjs#2902]
Here are some minor issues in the doc examples, that I have found.

# [quantileSeq](https://mathjs.org/docs/reference/functions/quantileSeq.html)
```
quantileSeq([3, -1, 5, 7], [1/3, 2/3])
TypeError: Unexpected type of argument in function quantileSeq
```
It seems the 2nd argument must not be an array. But accordig to the docs it should work.

# [to](https://mathjs.org/docs/reference/functions/to.html)
```
to(unit('2 inch'), unit(null, 'cm')) # returns Unit 5.08 cm
TypeError: Unexpected type of argument in function unit (expected: number or BigNumber or Fraction or Unit or string or Array or Matrix or Complex or boolean, actual: identifier | null, index: 0)
```
`unit(null, 'cm')` is strange. What should this do?

# [distance](https://mathjs.org/docs/reference/functions/distance.html)
in the final line of the examples the closing `)` is missing
```
distance(  {pointX: 2, pointY: 3, pointZ: 1},  {x0: 1, y0: 1, z0: 2, a: 5, b: 0, c: 1}
SyntaxError: Parenthesis ) expected (char 87)
```

# [getMatrixDataType](https://mathjs.org/docs/reference/functions/getMatrixDataType.html)
```
fractionX = [ [fraction(1, 3)], [fraction(1, 3] ]
SyntaxError: Parenthesis ) expected (char 47)
```
closing `)` of last `fraction()` missing. This is correct:
```
fractionX = [ [fraction(1, 3)], [fraction(1, 3)] ]
#                                             ^correct
```

# [max](https://mathjs.org/docs/reference/functions/max.html)
```
max([[2, 5], [4, 3]], [1, 7], 1) 
#                  ^ wrong
```
```
max([[2, 5], [4, 3], [1, 7] ], 1) 
#                           ^correct
```

# [subset](https://mathjs.org/docs/reference/functions/subset.html)
```typescript
  const e: any = [];
  const f = mathJs.subset(e, mathJs.index(0, [0, 2]), [5, 6]);
  /**
   * variable | according to the docs | actual 
   *    e     |          [[5, 0, 6]]  | []
   *    f     |          [[5, 6]]     | [ [ 5, 0, 6 ] ]
   * 
   */
```
Should `e` really be changed after calling `subset`?

**Comment by josdejong** [comment]
Thanks!!


### quantileSeq

This actually was a bug: the function `quantileSeq` did not accept a matrix as second argument, only an array. It would be helpful to rewrite the function as a `typed-function`, but for now I applied a small fix.

### to

If I remember well, `unit(null, 'cm')` was possible in the past to create a unit without value. Right now you have to do `unit('cm')` instead. I've fixed the example accordingly.

### distance

Fixed

### getMatrixDataType

Fixed 

### max

Fixed 

### subset

Fixed

**Comment by josdejong** [comment]
The fixes are now published in `v11.6.0`.
